mods/Other/formspecs/init.lua

```diff
@@ -14,12 +14,21 @@ minetest.FORMSPEC_SIGTERM = 3		-- server is shutting down
 minetest.FORMSPEC_SIGPROC = 4		-- procedural closure
 minetest.FORMSPEC_SIGTIME = 5		-- timeout reached
 
+local srng = SecureRandom()
+if srng ~= nil then
+	minetest.log("error", "Unable to initalize random number generator, formspecs may be insecure.")
```

Comment:
Shouldn't the check be `srng == nil`? Like, isn't the error all about `SecureRandom()` returning `nil`?

Reply:
Yeah it should be, my bad, I didn't check what i was writing well enough.

Reply:
probably should be `if not srng`. worst case its just nagging now